src/murmur/Meta.cpp

```diff
@@ -228,8 +228,13 @@ void MetaParams::read(QString fname) {
 			break;
 	}
 
-	qWarning("Initializing settings from %s (basepath %s)", qPrintable(qsSettings->fileName()),
-			 qPrintable(qdBasePath.absolutePath()));
+	if (QFile::exists(qsAbsSettingsFilePath)) {
+		qWarning("Initializing settings from %s (basepath %s)", qPrintable(qsSettings->fileName()),
+				 qPrintable(qdBasePath.absolutePath()));
+	} else {
+		qWarning("No ini file at %s (basepath %s). Initializing with default settings!",
```

Comment:
```suggestion
		qWarning("No ini file at %s (basepath %s). Initializing with default settings.",
```
I think we can save exclamation marks for actually important messages :)